WeakAuras/GenericTrigger.lua

```diff
@@ -3817,6 +3817,15 @@ WeakAuras.GetItemSubClassInfo = function(i)
   return GetItemSubClassInfo(classId, subClassId)
 end
 
+WeakAuras.GetCritChance = function()
+  -- Based on what the wow paper doll does
+  local spellCrit = GetSpellCritChance(2)
+  for i = 3, MAX_SPELL_SCHOOLS do
```

Comment:
MAX_SPELL_SCHOOLS does not exists on classic

Reply:
Oh, hrmpf, what does the PaperDollFrame.lua do on classic...